cypress/support/cypress-support/src/utils.js

```diff
@@ -337,29 +337,34 @@ function unsubscribe(webSocketClient = null) {
  * isScenarioExempted("advertising.setSkipRestriction");
  **/
 function isScenarioExempted(method, param) {
-  let isInList = false;
```

Comment:
Returning a key name of exception list Ex: "NOT_SUPPORTED_METHODS",  if the method and param present in the list. Before we returning boolean when it is present.